pyproject.toml

```
@@ -1,3 +1,20 @@
 [build-system]
 requires = ["setuptools>=61.0.0", "wheel"]
-build-backend = "setuptools.build_meta"
\ No newline at end of file
+build-backend = "setuptools.build_meta"
+
+[tool.mypy]
+strict = "True"
+exclude = ["setup.py"]
```

I would like to exclude a possible 'venv' folder
```suggestion
exclude = ["setup.py", "venv"]
```